docs/cub/benchmarking.rst

```diff
@@ -229,6 +229,34 @@ run the benchmark regardless, and continue.
 The tuning infrastructure stores results in an SQLite database called `cccl_meta_bench.db` in the build directory.
 This database persists across tuning runs.
 If you interrupt the benchmark script and then launch it again, only missing benchmark variants will be run.
+
+
+Comparing results of multiple tuning databases
+--------------------------------------------------------------------------------
+
+Similarly to comparing JSON files from NVBench,
+benchmark results captured in a tuning database can be compared as well:
+
+.. code-block:: bash
+
+    ../benchmarks/scripts/compare.py -o cccl_meta_bench1.db cccl_meta_bench2.db
+
+This will print a Markdown report showing the runtime differences and noise for each variant.
+
+Furthermore, you can plot the results from one or more tuning databases as bar chart or box plot (add `--box`):
```

Comment:
Question: Not a native speaker but shouldn't this either be `as a bar plot` or `as bar plots` because databases is plural?

Reply:
Agreed. Since we plot multiple databases as a single plot, let's add an article.